Troubleshoot/TroubleshootError.ps1

```diff
@@ -50,18 +50,6 @@ $azureRmAks = Get-Module -ListAvailable -Name AzureRM.AKS;
 
 if (($null -eq $azureRmProfileModule) -or ($null -eq $azureRmResourcesModule) -or ($null -eq $azureRmOperationalInsights) -or ($null -eq $azureRmAks)) {
 
-    $currentPrincipal = New-Object Security.Principal.WindowsPrincipal([Security.Principal.WindowsIdentity]::GetCurrent())
```

Comment:
this is the specific bit that fails on mac / cloudshell

Reply:
@timja Yes we will look into it.